### PR TITLE
some fixes for cuda easyblock for CUDA SDK 7.5

### PR DESCRIPTION
**Pull request by molden**
This PR depends on ~~https://github.com/hpcugent/easybuild-framework/pull/1417~~ .

### CODE REVIEW

File: easybuild/easyblocks/c/cuda.py [hunk 1]
```diff
@@ -89,8 +89,7 @@ def sanity_check_step(self):
         custom_paths = {
             'files': ["bin/%s" % x for x in ["fatbinary", "nvcc", "nvlink", "ptxas"]] +
                      ["%s/lib%s.so" % (x, y) for x in chk_libdir for y in ["cublas", "cudart", "cufft",
-                                                                           "curand", "cusparse"]] +
-                     ["open64/bin/nvopencc"],
```
Comment (boegel) [inline easybuild/easyblocks/c/cuda.py]: so, this is no longer around for recent versions? we should only drop this for recent versions then?

something like this:

```python
extra_files = []
if LooseVersion(self.version) < LooseVersion('7.5'):
    extra_files.append('open64/bin/ncopencc')

custom_paths = {
    ...
```

and then use `extra_files` in here?

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyblocks/c/cuda.py [hunk 2]
```diff
@@ -75,7 +75,7 @@ def install_step(self):
         if 'DISPLAY' in os.environ:
             os.environ.pop('DISPLAY')
 
-        run_cmd_qa(cmd, qanda, std_qa=stdqa, no_qa=noqanda, log_all=True, simple=True)
+        run_cmd_qa(cmd, qanda, std_qa=stdqa, no_qa=noqanda, log_all=True, simple=True, maxhits=1500)
```
Comment (boegel) [inline easybuild/easyblocks/c/cuda.py]: this allows the installer to hang without producing new output for 1500 seconds before EB gives up... isn't that *very* long?

maybe it's just a matter of fleshing out the `noqanda` a bit?

how did you see it hang exactly?

File: easybuild/easyblocks/c/cuda.py (hunk 1 above)
Comment (boegel) [inline easybuild/easyblocks/c/cuda.py]: OK, covered now, thanks!

File: easybuild/easyblocks/c/cuda.py (hunk 2 above)
Comment (molden) [inline easybuild/easyblocks/c/cuda.py]: Well it produces indeed for a long time no output apparently. But I will have a look to reduce it a little a bit with your suggestion.

Comment (boegel) [inline easybuild/easyblocks/c/cuda.py]: ping me when it's tackled, and I'll test it with existing CUDA easyconfigs + https://github.com/hpcugent/easybuild-easyconfigs/issues/2039

Comment (molden) [inline easybuild/easyblocks/c/cuda.py]: Actually I think it was yesterday a local problem. Access to storage was very slow temporarily. I guess we can reduce it to let's say 300?

Comment (boegel) [inline easybuild/easyblocks/c/cuda.py]: Sure, just add a comment maybe why you're overriding the default value, and what is means (300s wait for nothing to change in the output without seeing a known question)

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyblocks/c/cuda.py [hunk 3]
```diff
@@ -86,11 +87,14 @@ def sanity_check_step(self):
         if LooseVersion(self.version) < LooseVersion("6"):
             chk_libdir += ["lib"]
 
+        extra_files = []
+        if LooseVersion(self.version) < LooseVersion('7'):
+            extra_files.append('open64/bin/ncopencc')
```
Comment (boegel) [inline easybuild/easyblocks/c/cuda.py]: typo: should be `nvopencc`, not `ncopencc` (yaay for testing)

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
good to go, retested will existing CUDA easyconfigs + the one in https://github.com/hpcugent/easybuild-easyconfigs/pull/2039, see https://github.com/hpcugent/easybuild-easyconfigs/pull/2039#issuecomment-146819673

**Comment by boegel** [comment]
Thanks @molden!

**Pull request merged**